### ISSUE TITLE
Telecom Dataset Analysis

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/ML-Crate#468]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** : Telecom Dataset Analysis
:red_circle: **Aim** : Create a bunch of ML models to analyze the telecom dataset.
:red_circle: **Dataset** : https://www.kaggle.com/datasets/aesedeu/telecom-dataset
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Participant ID (If not, then put NA) :
- Approach for this Project :
- What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Yuvika-14** [comment]
hi abhisekh
Full name: Yuvika Singh
git hub profile link:https://github.com/Yuvika-14/Yuvikademo.git
approch: ensemble methods, gradient boosting, neural networks.
Take care of the missing data if there are any categorical values then will use one hot encoder or label encoder depending upon the issue.
Seeking forward for your response

**Comment by CodexAbhi** [comment]
name: Abhimanyu Jaiswal
github profile: https://github.com/CodexAbhi
participant id: NA
approach: research about the data, as there are no missing values, just check for outliers and plot some graphs and check relation of different features in exploratory data analysis. use some regression models and do hyperparameter tuning to improve score, going to try fitting dense neural network also if it gives good results, proceed on that.
participant role: IWOC

**Comment by abhisheks008** [comment]
@CodexAbhi @Yuvika-14 this is part of JWOC

**Comment by pawaspy** [comment]
@abhisheks008 in this telecom dataset is in russian language. It might be difficult for me and others to understand what are those columns saying can you add another.

**Comment by abhisheks008** [comment]
Hi @pawaspy thanks for the information. I'll fix this for sure.

**Comment by pawaspy** [comment]
I wanted to work under this project but due to different language in the dataset it will be hard for me to detect. Or I can do is convert the foreign language into English and then perform the task.If thats ok can you assign me this task under IWoC?

**Comment by abhisheks008** [comment]
Will update on this.

**Comment by webarebears9** [comment]
Full name : Tulika Bhatia
GitHub Profile Link : https://github.com/webarebears9
Participant ID (If not, then put NA) :NA
Approach for this Project : decision tree, random forest,linear regression, polynomial regression 
Participant role : SSOC

**Comment by abhisheks008** [comment]
> Full name : Tulika Bhatia GitHub Profile Link : https://github.com/webarebears9 Participant ID (If not, then put NA) :NA Approach for this Project : decision tree, random forest,linear regression, polynomial regression Participant role : SSOC Seeking forward for your response.

Use the following models for this project.
- Random Forest
- Decision Tree
- Linear Regression
- Gradient Boosting
- XGBoost
- Lasso
- Ridge
- Polynomial regression

Assigned @webarebears9
